### DISCUSSION TITLE
Supporting react-native 0.69 roadmap

### DISCUSSION THREAD

**Discussion by dratwas** [react-native-tvos/react-native-tvos#347] [General Discussions]
Hi, first of all, i wanted to thank you for your hard work on that library. It's awesome! 

I have a question. We have set up a project with react-native 0.69 and will need to support tvOS soon. Do you have the roadmap to move to 0.69? I’m in no way trying to ask when a 0.69 react-native-tvos will be released. Just wanted to know if you already started working on that. We could try it in our app, test it, and report/fix issues if any ;)

**Comment by douglowder** [comment]
I have indeed started work on this. I hope to release soon.

**Comment by douglowder** [comment]
aaaaaand it has been published @dratwas
